### ISSUE TITLE
Fix No Artifact Upgrade Selected On Prestige

### ISSUE THREAD

**Issue by becurrie** [becurrie/titandash#480]
Currently, if the artifact upgrade settings are unable to derive an artifact to upgrade on a prestige, an exception is raised since the image is None, which breaks our `cv2.imread` functionality when searching for the artifact.

```
cv2.error: OpenCV(4.1.1) C:\projects\opencv-python\opencv\modules\imgproc\src\color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cv::cvtColor'
```

**Comment by skvie** [comment]
Fixed with https://github.com/becurrie/titandash/pull/479

**Comment by becurrie** [comment]
Resolved by #488
